packages/schematics/angular/ng-new/index_spec.ts

```diff
@@ -46,4 +46,18 @@ describe('Ng New Schematic', () => {
     const content = tree.readContent('/bar/angular.json');
     expect(content).toMatch(/"prefix": "pre"/);
   });
+
+  it('should set strict options in tsconfig.json', () => {
```

Comment:
This test should live in the workspace schematic, because that is where the logic exists.

Reply:
Are you sure? I put it there on purpose, because even if the logic is in the workspace, the user will use this option from `ng new`, so putting it there also verify that the option is passed well (to avoid problems like in my initial PR where I missed the passing logic). Or maybe do the test twice?